apps/mobile/src/app/wallet/settings/display/index.tsx

```diff
@@ -40,7 +43,7 @@ export default function SettingsDisplayScreen() {
       >
         <Cell
           title={t`Theme`}
-          subtitle={t`System`}
+          subtitle={capitalize(settings.themeStore)}
```

Comment:
I added the `capitalize` util to our utils pkg bc there are times we don't always want to do a text transform to capitalize the string passed. Here, the subtitle is often all cap tickers.

Reply:
I'm not sure we need to do this - https://github.com/leather-io/mono/pull/427/files#r1744823802

Reply:
I don't think we can say we'll never need it, on grounds we should prefer css capitalisation. Working with external libraries for example where you can't access css. 

Either way, this is now no longer localised? How do we add a conditional, dynamic localised string?

Reply:
Thanks @fbwoolf 👍 no problem using it, I was just wondering why

Reply:
@kyranjamie it appears you have to use the `useLingui` hook for translating variables...

```ts
const { i18n } = useLingui();
i18n._(capitalize(themeStore))
```